app/src/pages/GuestQuestions/GuestQuestions.tsx

```diff
@@ -0,0 +1,96 @@
+import React from 'react'
```

Comment:
looks like your demo files for formik got in here - can you delete them out of this branch?

Reply:
Well do when I get home

Reply:
so i checked everything
![Screenshot (39)](https://user-images.githubusercontent.com/48224444/82174717-67a30d00-9886-11ea-9b78-2a66de181550.png)

Reply:
i also have the prettierrc file as well